src/dxgi/dxgi_output.cpp

```diff
@@ -637,4 +621,51 @@ namespace dxvk {
     }
   }
 
+
+  void DxgiOutput::CacheMonitorData() {
+    // Try and find an existing monitor info.
+    {
```

Comment:
again weird random scope that seems to exist for no reason

Reply:
Fixed.